webapp/channels/src/i18n/nl.json

```diff
@@ -4385,16 +4385,13 @@
   "postypes.custom_open_pricing_modal_post_renderer.upgrade_professional": "Upgrade naar Professional",
   "postypes.custom_open_pricing_modal_post_renderer.userRequests": "{userRequests} vroegen toegang tot deze functie",
   "postypes.custom_open_pricing_modal_post_renderer.view_options": "Bekijk de upgrade-opties",
-  "pricing_modal.addons.247Coverage": "24x7 bereikbaar",
```

Comment:
We generally dont remove or edit other languages as they are automatically updated from Weblate

Reply:
For edit, they will be updating the texts but does it hurt to delete ? Haven't had issues in the past

Reply:
For either edit or deletion, it gets synchronised in weblate

Reply:
Agreed, so if all are deleted then it doesn't hurt to have had them removed